### DISCUSSION TITLE
CUDA Error in EarlyStopping

### DISCUSSION THREAD

**Discussion by macabdul9** [Lightning-AI/pytorch-lightning#8306] [General]
Getting this error during evaluation. @tchaton 

    
![Screenshot from 2021-07-06 13-59-18](https://user-images.githubusercontent.com/25720695/124568563-acab4980-de62-11eb-982b-d6a164c024e3.png)

**Comment by Borda** [comment]
@macabdul9 I feel this is rather an issue, and so mind share some sample code to reproduce?
